src/libs/auth.js

```diff
@@ -169,8 +169,8 @@ authStore.subscribe(withErrorReporting('Error checking registration', async (sta
     try {
       const { enabled } = await Ajax().User.getStatus()
       if (enabled) {
-        // Once the ToS grace period has ended in production, if Sam returns `enabled` above, `acceptedTos` should also be true.
-        // At that point, we can add an assertion here.
+        // While initial state is first loading, state.acceptedTos will be undefined (it will then be `true` on the
```

Comment:
Updating the comment because we can't add the assertion-- this code is executed a couple of times while initial state is getting sorted out, and the first time `acceptedTos` is undefined.